Laravel.gitignore

```diff
@@ -1,5 +1,7 @@
 vendor/
 node_modules/
+.rocketeer/
```

Comment:
Maybe I don't understand the system, but afaik, we don't use different .gitignore files in one folder. So covering some deployment systems in this file is a good way. Or am I wrong?